File: 191-number-of-1-bits/solution.cpp
// Runtime: 0ms, Beats 100.00% of users with C++ 
// Memory: 7.02MB, Beats 35.70% of users with C++


class Solution {
public:
    int hammingWeight(uint32_t n) {
        return __builtin_popcount(n);
    }
};

File: 191-number-of-1-bits/solution.py
# Runtime: 37ms, Beats 53.54% of users with Python3
# Memory: 16.47MB, Beats 90.89% of users with Python3

class Solution:
    def hammingWeight(self, n: int) -> int:
        binary=bin(n)
        bits=[bit for bit in binary if bit == '1']
        return len(bits)